pages/requests/[uuid].js

```diff
@@ -29,13 +29,17 @@ import {
 const Request = () => {
   const router = useRouter()
   const { data: session } = useSession()
-  // parses the query string to get the quote group (request) id
-  const { id } = router.query
-  const { request, isLoadingRequest, isRequestError } = useOneRequest(id, session?.accessToken)
-  const { allSOWs, isLoadingSOWs, isSOWError } = useAllSOWs(id, request?.identifier, session?.accessToken)
-  const { messages, isLoadingMessages, isMessagesError, mutate, data } = useMessages(request?.uuid, session?.accessToken)
-  const { files, isLoadingFiles, isFilesError } = useFiles(id, session?.accessToken)
+  /**
+   * since this is a dynamically routed file, the router query will always consist of a "key: value" pair that's determined by the name of
+   * the file (key) and path string (value). additional query properties may also exist if they were explicitly passed.
+  */
```

Comment:
this file was renamed for this reason.

all references to `id` in the functions below were then updated to `uuid`.

Reply:
```suggestion
  /**
   * as a dynamically routed file, the router query will always consist of a "key: value" pair that's determined by the name of
   * the file (key) and path string (value). additional query properties may also exist if they were explicitly passed.
  */
```